src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -1501,12 +1526,27 @@ internal void UninstallPackage(string packageId)
                         action,
                         CancellationToken.None);
                 },
-                nugetUi => SetOptions(nugetUi, NuGetActionType.Uninstall));
+                nugetUi => SetOptions(nugetUi, NuGetActionType.Uninstall, new[] { package }));
         }
 
+        /// <summary>
+        /// For Apex tests
```

Comment:
Please add specific details about what change applies to Apex tests, and what the original problem was.

Reply:
Removed specia Apex Methods.

Reply:
Removed.